src/stylesheet-registry.js

```diff
@@ -163,11 +183,17 @@ function invariant(condition, message) {
   }
 }
 
-export const StyleSheetContext = createContext(new StyleSheetRegistry())
+export const StyleSheetContext = createContext(null)
+
+export function createStyleRegistry() {
+  return new StyleSheetRegistry()
+}
 
-export function StyleRegistry({ children }) {
+export function StyleRegistry({ registry: configuredRegistry, children }) {
   const rootRegistry = useContext(StyleSheetContext)
-  const [registry] = useState(() => rootRegistry || new StyleSheetRegistry())
+  const [registry] = useState(
+    () => configuredRegistry || rootRegistry || createStyleRegistry()
```

Comment:
Since we only want 1 registry to be created globally, shouldn’t rootRegistry always take priority here?